board_orders.md

```diff
@@ -126,3 +126,16 @@ In the event the Fraternity makes a donation or other contribution to a non-prof
     * During that meeting, all elected positions will be discussed in order, as described by the Bylaws.
     * Each candidate of the current position will have 5 minutes to give a speech.
     * After each speech is given, all candidates of the position must leave the room and the rest of the membership will have 5 minutes to talk about the candidates.
+    
+## Board Order V. Risk Management Duties
+
+The following is a list of things that all RMs should look out for/do while on duty:

```

Comment:
We should clarify the definition of who we're talking about before the list.

For example:

> One or more Risk Management officers or their previously announced designee should be present and completely sober at all official LAN events. For the purposes of this Order, an "Acting Risk Manager" is this officer or their previously-announced designee.